packages/components/select/src/useSelect.ts

```diff
@@ -445,11 +444,7 @@ export const useSelect = (props: ISelectProps, emit) => {
       }
     }
     if (option) return option
-    const label = isObjectValue
-      ? value.label
-      : !isNull && !isUndefined
-      ? value
-      : ''
+    const label = isObjectValue ? value.label : !isNullish ? value : ''
```

Comment:
Or just inline isNullish by using nullish coalescing operator?
```suggestion
    const label = isObjectValue ? value.label : value ?? ''
```

Reply:
Good catch